doc/reference/misc.md

```diff
@@ -6225,3 +6244,178 @@ if `limit` is not false.
+with positive and negative infinity, compactifying their order.
+Positive infinity is represented by boolean `#f` while
+negative infinity is represented by boolean `#t`.
+Common arithmetic operations are provided, partially defined,
+and raising an error in unsupported cases such as adding opposite infinites or dividing by zero.
+
+::: tip To use the bindings from this module:
+``` scheme
+(import :std/misc/xreal)
+```
+:::
+### xreal?
+``` scheme
+(xreal? obj) -> boolean
+```
+
+`xreal?` returns a boolean that is true if the object represents either a real or a boolean,
+where booleans `#f` and `#t` respectively represent positive and negative infinity.
+
+### xreal<
+``` scheme
+(xreal< <x1> ... <xn>) -> boolean
+```
+
+`xreal<` returns a boolean that is true if the list of its extended real arguments
+is strictly increasing.
+
+### xreal</list
+``` scheme
+(xreal</list <l>) -> boolean
+```
+
+`xreal<` returns a boolean that is true if its argument, a list of extended real numbers,
+is strictly increasing.
+
+### xreal<=
+``` scheme
+(xreal<= <x1> ... <xn>) -> boolean
+```
+
+`xreal<=` returns a boolean that is true if the list of its extended real arguments
+is non-decreasing.
+
+### xreal<=/list
+``` scheme
+(xreal<=/list <l>) -> boolean
+```
+
+`xreal<=/list` returns a boolean that is true if its argument, a list of extended real numbers,
+is non-decreasing.
+
+### xreal-min
+``` scheme
+(xreal-min <x1> ... <xn>) -> xreal
+```
+
+`xreal-min` returns the lower bound of the set of its extended real arguments.
+In particular, it returns `#f` (representing the positive infinity) if provided zero arguments,
+and is the identity function when given a single argument.
+
+### xreal-min/list
+``` scheme
+(xreal-min/list <l>) -> xreal
+```
+
+`xreal-min/list` returns the lower bound of the list of extended real arguments passed as its arguments.
+In particular, it returns `#f` (representing the positive infinity) if provided an empty list.
+
+### xreal-min!
+``` scheme
+(xreal-min! <var> <x> ...) -> void
+```
+
+`xreal-min!` side-effects a variable to change it to the `xreal-min`
+of the previous value and the provided arguments.
+
+### xreal-max
+``` scheme
+(xreal-max <x1> ... <xn>) -> xreal
+```
+
+`xreal-max` returns the upper bound of the set of its extended real arguments.
+In particular, it returns `#f` (representing the posinegative infinity) if provided zero arguments,
+and is the identity function when given a single argument.
+
+### xreal-max/list
+``` scheme
+(xreal-max/list <l>) -> xreal
+```
+
+`xreal-max/list` returns the lower bound of the list of extended real arguments passed as its arguments.
+In particular, it returns `#t` (representing the negative infinity) if provided an empty list.
+
+### xreal-max!
+``` scheme
+(xreal-max! <var> <x> ...) -> void
+```
+
+`xreal-max!` side-effects a variable to change it to the `xreal-max`
+of the previous value and the provided arguments.
+
+
+(def xreal-max
```

Comment:
why are these defs in the documentation?

Reply:
Oops, I was in the middle of writing it when I attended to other issues and was interrupted... fixing that...